README.md

```diff
@@ -1,95 +1,63 @@
+# Advance Valuation Rulings UI Tests
 
-# advance-valuation-rulings-ui-tests
 UI test suite for the `advance-valuation-rulings-frontend` using Selenium WebDriver and Cucumber.  
 
 ---
 
-### Running tests locally against a containerised browser - on a developer machine
+## Starting Services
 
-Prior to executing the tests ensure you have:
- - Docker - to run mongo and browser (Chrome, Firefox or Edge) inside a container 
- - Appropriate [drivers installed](#installing-local-driver-binaries) - to run tests against locally installed Browser
- - Installed/configured [service manager](https://github.com/hmrc/service-manager).  
+To start the required services via [service manager](https://github.com/hmrc/sm2), run:
 
-Run the following command to start services locally:
-
-```
-docker run --rm -d --name mongo -d -p 27017:27017 mongo:5.0
-```
-
-```
+```shell
```

Comment:
change `shell` to `bash` -> bash is used as default in jenkins so locally we can try to have the same

Reply:
I would rather leave this one as shell and the other script ones as bash. This is what is used in the [sm2](https://github.com/hmrc/sm2/blob/main/README.md?plain=1#L77) readme. I have been updating mine to shell for the sm2 stuff.